components/pages/explorer/Jbrowse/JbrowseLaunchButton.tsx

```diff
@@ -19,64 +19,145 @@
  *
  */
 
-import Button from '@/components/Button';
+import { getDropdownTheme } from '@/components/theme/getDropdownTheme';
+import { Spinner } from '@/components/theme/icons';
 import { CustomTooltip } from '@/components/Tooltip';
 import { css, useTheme } from '@emotion/react';
+import { TransparentButton } from '@overture-stack/arranger-components/dist/Button';
+import { MultiSelectDropDown } from '@overture-stack/arranger-components/dist/DropDown';
 import { find } from 'lodash';
-import { ButtonWrapper } from '../ActionBar';
-import { RepositoryTabNames } from '../RepositoryContent';
 import { useTabsContext } from '../TabsContext';
+import { RepositoryTabKeys, RepositoryTabNames } from '../types';
 import useJbrowseCompatibility from './useJbrowseCompatibility';
+import { jbrowseDict, JbrowseTitles } from './utils';
 
 const JbrowseLaunchButton = () => {
   const theme = useTheme();
-  const { handleChangeTab, handleOpenTab, openTabs } = useTabsContext();
-  const { jbrowseEnabled, jbrowseErrorText, jbrowseLoading } = useJbrowseCompatibility();
+  const { handleChangeTab, handleUpdateTab, handleOpenTab, openTabs } = useTabsContext();
+  const {
+    jbrowseCircularEnabled,
+    jbrowseLinearEnabled,
+    jbrowseCircularError,
+    jbrowseLinearError,
+    jbrowseLoading,
+  } = useJbrowseCompatibility();
+
+  const handleJbrowseSelect = (
+    jbrowseOptionKey: RepositoryTabKeys,
+    closeDropDownFn: () => void,
+  ) => {
+    const alternateJbrowseTab = find(openTabs, { name: RepositoryTabNames.GENOME_VIEWER });
+    if (find(openTabs, { key: jbrowseOptionKey })) {
+      // if selected option has a tab open, go to that tab
+      handleChangeTab(jbrowseOptionKey);
+    } else if (alternateJbrowseTab) {
+      // update existing jbrowse tab if available - only 1 jbrowse tab open at a time
+      handleUpdateTab(alternateJbrowseTab.key, { key: jbrowseOptionKey });
+      handleChangeTab(jbrowseOptionKey);
+    } else {
+      handleOpenTab({
+        name: RepositoryTabNames.GENOME_VIEWER,
+        key: jbrowseOptionKey,
+        canClose: true,
+      });
+    }
+    closeDropDownFn();
+  };
+
+  const dropdownTheme = getDropdownTheme(theme);
 
   return (
-    <CustomTooltip
-      disabled={jbrowseEnabled}
-      unmountHTMLWhenHide
-      arrow
-      html={
+    <div
+      css={css`
+        .genome-viewer-dropdown * {
+          // stops scrollbar on dropdown
+          // positions tooltip properly
+          overflow: hidden !important;
+        }
+      `}
+    >
+      <MultiSelectDropDown
+        theme={{
+          ...dropdownTheme,
+          width: '140px',
+          height: '30px',
+          ListWrapper: {
+            ...dropdownTheme.ListWrapper,
+            css: css`
+              left: -2px;
+              right: auto;
+              width: 7em;
+            `,
+          },
```

Comment:
Fix to the need to "left: -2; right: auto", I believe 🤞 
```suggestion
      <MultiSelectDropDown
        align="left"
        theme={{
          ...dropdownTheme,
          width: '140px',
          height: '30px',
          ListWrapper: {
            ...dropdownTheme.ListWrapper,
            css: css`
              width: 7em;
            `,
          },
```